stable/testlink/values.yaml

```diff
@@ -162,6 +162,7 @@ service:
   ## ref http://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip
   ##
   externalTrafficPolicy: Cluster
+  annotations: {}
```

Comment:
Once you implement the changes I requested before, add this comment:
```yaml
  ## Provide any additional annotations which may be required. 
  ## Evaluated as a template
  ##
  annotations: {}
```

Reply:
Please also update the parameter's description in the README.md so it's sth like: `Kubernetes service annotations, evaluated as a template`

Reply:
done